integration/test/Testlib/Types.hs

```diff
@@ -202,7 +202,9 @@ data ServiceOverrides = ServiceOverrides
     dbNginz :: Value -> App Value,
     dbSpar :: Value -> App Value,
     dbBackgroundWorker :: Value -> App Value,
-    dbStern :: Value -> App Value
+    dbStern :: Value -> App Value,
+    dbFederatorInternal :: Value -> App Value,
+    dbFederatorExternal :: Value -> App Value
```

Comment:
Did this get used by this PR? If not, please remove it otherwise this conflicts with some work I am doing in #3450